### PR TITLE
Doc: add HelloTalk into powered-by

### PR DESCRIPTION
**Pull request by lilien1010**
### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

**Comment by juzhiyuan** [comment]
What about this PR https://github.com/apache/incubator-apisix/pull/690

**Comment by lilien1010** [comment]
@juzhiyuan  I close that one ,since that branch patch-3   is outdate

File: doc/powered-by.md [hunk 1]
```diff
@@ -48,3 +49,6 @@ Using APISIX as a gateway, it uses the functions of current limiting, speed limi
 
 ## souche.com
 Using APISIX as a Web ACL gateway to deal with backend OA systems traffic.
+
+## HelloTalk, Inc
+Using APIS as a api gateway to manager all API and SSL Certificate in test\dev\CMS environment.
```
Comment (moonming) [inline doc/powered-by.md]: typo, should be `Apache APISIX`

File: doc/powered-by.md [hunk 2]
```diff
@@ -51,4 +51,4 @@ Using APISIX as a gateway, it uses the functions of current limiting, speed limi
 Using APISIX as a Web ACL gateway to deal with backend OA systems traffic.
 
 ## HelloTalk, Inc
-Using APIS as a api gateway to manager all API and SSL Certificate in test\dev\CMS environment.
+Using Apache APISIX as a API gateway to manager all API and SSL Certificates in test\dev\CMS environment.
```
Comment (juzhiyuan) [inline doc/powered-by.md]: `an API`

Comment (juzhiyuan) [reply doc/powered-by.md]: `to manage`

**Pull request merged**